File: test_archives/py_tests/batch2/test_function0.py
from functions.function0 import function0
import pytest

def test_function0_1(): 
    assert function0(1, 2.3, "four", [5, 6]) == 9.3

def test_function0_2(): 
    assert function0(2, 4.6, "eight", [10, 12]) == 13.6

File: test_archives/cpp_tests/batch2/test_function0.cpp
#include "../catch.hpp"
#include "../functions/function0.cpp"

TEST_CASE("Test function0_1", "[function0]") {
    REQUIRE(function0(1, 2.3, "four", {5, 6}) == 9.3);
}

TEST_CASE("Test function0_2", "[function0]") {
    REQUIRE(function0(2, 4.6, "eight", {10, 12}) == 13.6);
}